### PR TITLE
Allow getting the list of updates in JSON format from fwupdagent

### PR DESCRIPTION
**Pull request by hughsie**
Apparently this is wanted by the SOS team too.

### CODE REVIEW

File: data/bash-completion/fwupdagent [hunk 1]
```diff
@@ -1,5 +1,6 @@
 _fwupdagent_cmd_list=(
 	'get-devices'
+	'get-updates'
```
Comment (superm1) [inline data/bash-completion/fwupdagent]: Missing get upgrades

File: src/fu-agent.c [hunk 2]
```diff
@@ -113,6 +155,50 @@ fu_util_get_devices (FuUtilPrivate *priv, gchar **values, GError **error)
...
+	}
+
+	/* create header */
+	builder = json_builder_new ();
+	json_builder_begin_object (builder);
+	if (!fu_util_add_updates_json (priv, builder, error))
+		return FALSE;
+	json_builder_end_object (builder);
+
+
```
Comment (superm1) [inline src/fu-agent.c]: Extra new line

File: data/bash-completion/fwupdagent (hunk 1 above)
Comment (hughsie) [reply data/bash-completion/fwupdagent]: Fixed.

File: src/fu-agent.c (hunk 2 above)
Comment (hughsie) [reply src/fu-agent.c]: Fixed, thanks.

**Pull request merged**